kafka/base.py

```diff
@@ -127,7 +127,7 @@ def latest_offset(self):
         """
         raise NotImplementedError
 
-    def earliest_offset(self):
+    def earliest_available_offset(self):
```

Comment:
Big +1 to name changes.  Irrespective of the needs of the Offset CF API, these are much more descriptive names.